pkg/kubelet/kubelet_node_status.go

```diff
@@ -155,6 +160,22 @@ func (kl *Kubelet) tryRegisterWithApiServer(node *v1.Node) bool {
 	return false
 }
 
+// reconcileLablesWithExistingNode reconciles labels on a new node and the existing node, returning
+// whether the existing node must be updated.
+// Current reconcile strategy: always use labels on new node.
+func (kl *Kubelet) reconcileLablesWithExistingNode(node, existingNode *v1.Node) bool {
```

Comment:
This function name is misleading. There's no reconciliation here.

The function name could more accurately be called "clobberAnyUserConfigurationChanges". It's not reconciling.